backend/app/services/adapters/task_kinds/queries.py

```
@@ -33,6 +33,30 @@
+        task_id: Task ID for logging purposes
+        task_json: Task JSON data (expected to be a dict)
+
+    Returns:
+        True if is_group_chat is explicitly True
+        False if is_group_chat is explicitly False or not present
+        None if parsing failed (malformed data)
+    """
+    try:
+        if isinstance(task_json, dict):
+            return task_json.get("spec", {}).get("is_group_chat") is True
+        return False
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)